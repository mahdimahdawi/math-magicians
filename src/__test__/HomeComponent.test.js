import { render, screen } from '@testing-library/react';
import Home from '../components/home';

describe('Testing Home Component', () => {
  test('testing home', () => {
    render(<Home />);

    // eslint-disable-next-line testing-library/await-async-query
    expect(screen.findByText('Welcome To Our Page !')).toMatchSnapshot();
  });
});