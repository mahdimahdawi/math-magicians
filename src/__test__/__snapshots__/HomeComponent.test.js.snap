// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Home Component testing home 1`] = `Promise {}`;
